CONTRIBUTING.txt

```diff
@@ -60,13 +60,16 @@ Here's the long and short of it:
       git checkout main
       git pull upstream main
 
-   * Create a branch for the feature you want to work on. Since the
-     branch name will appear in the merge message, use a sensible name
+   * Create a branch for the feature you want to work on. Use a sensible name,
      such as 'transform-speedups'::
 
       git checkout -b transform-speedups
 
-   * Commit locally as you progress (``git add`` and ``git commit``)
+   * Commit locally as you progress (with ``git add`` and ``git commit``).
+     Please write `good commit messages
+     <https://vxlabs.com/software-development-handbook/#good-commit-messages>`_:
+     Capitalize the message, limit it to 50 characters, write it in the
+     imperative mood, etc.
```

Comment:
```suggestion
```

I suggest removing this; it can all be a bit overwhelming, and it is nicely summarized in that blog post (most English-speaking people have to look up imperative mood!).  It's also a bit subtle, because this really only applies to the subject line and not the rest of the commit message.

Reply:
Ok. In my intention, it was friendlier to expand a little in the body text, but I'm happy to save a couple of lines!